Libraries/oneMKL/monte_carlo_european_opt/mc_european.cpp

```diff
@@ -24,22 +24,31 @@
 namespace oneapi {}
 using namespace oneapi;
 
+// Temporary code for beta08 compatibility. Reduce routine is moved from intel::
+// to ONEAPI:: namespace
+#if __SYCL_COMPILER_VERSION < 20200902L
+using sycl::intel::reduce;
+#else
+using sycl::ONEAPI::reduce;
+#endif
+
 // Default number of options
-#define N_OPT  2048
+static const auto n_opt_default = 2048;
 // Default number of independent samples
-#define N_SAMPLES 262144
+static const auto n_samples_default = 262144;
 
 // Initialization value for random number generator
-#define SEED        7777
```

Comment:
in general, the variable names are not very descriptive and make it difficult to track

Reply:
Hi @JoeOster, SEED is a common name for random number generators usage, I have followed the same approach for variables naming as in [the classic MKL cookbook sample](https://software.intel.com/content/www/us/en/develop/documentation/mkl-cookbook/top/using-the-monte-carlo-method-for-simulating-european-options-pricing.html#using-the-monte-carlo-method-for-simulating-european-options-pricing)

Reply:
I wasn't being specific with SEED, rather was just a general comment about variables throughout.

Reply:
ok, fixed most variables names in Monte Carlo european options sample.